app/models/ta.rb

```diff
@@ -29,6 +29,28 @@ def get_num_marked(assignment)
     n
   end
 
+  def get_num_annotations(assignment)
```

Comment:
Okay, so the three methods above - make an optional parameter by a ta_id, which filters for a specific TA.